gradio/blocks.py

```diff
@@ -225,14 +225,6 @@ def get_config(self):
     def update(**kwargs) -> dict:
         return {}
 
-    @classmethod
```

Comment:
Looks like this was added here: https://github.com/gradio-app/gradio/pull/2309

The repros in the issue still work so I think we're ok to delete.